projects/1.16/assets/forestry/forestry/lang/zh_cn.json

```
@@ -0,0 +1,1505 @@
+    "forestry.allele.tolerance.both_3": "全3",
+    "forestry.allele.tolerance.both_4": "全4",
+    "forestry.allele.tolerance.both_5": "全5",
+    "forestry.allele.tolerance.down_1": "降1",
+    "forestry.allele.tolerance.down_2": "降2",
+    "forestry.allele.tolerance.down_4": "降4",
+    "forestry.allele.tolerance.down_5": "降5",
+    "forestry.allele.tolerance.none": "无",
+    "forestry.allele.tolerance.up_1": "加1",
+    "forestry.allele.tolerance.up_2": "加2",
+    "forestry.allele.tolerance.up_3": "加3",
+    "forestry.allele.tolerance.up_4": "加4",
+    "forestry.allele.tolerance.up_5": "加5",
+    "forestry.allele.lifespan.shortest": "早夭",
+    "forestry.allele.lifespan.shorter": "较短寿",
+    "forestry.allele.lifespan.short": "短寿",
+    "forestry.allele.lifespan.shortened": "不足寿",
+    "forestry.allele.lifespan.normal": "普寿",
+    "forestry.allele.lifespan.elongated": "足寿",
+    "forestry.allele.lifespan.long": "长寿",
+    "for.gui.size": "大小",
+    "for.gui.slots": "%1$s/%2$s 格子",
+    "for.gui.species": "品种",
+    "for.gui.speciescount": "种族",
+    "for.gui.speed": "工作速度",
+    "for.gui.stored": "已储存",
+    "for.gui.temperature": "温度",
+    "for.gui.tolerance": "适应性",
+    "for.gui.temptol": "温度适应性",
+    "for.gui.tooltip.tmi": "按住shift来查看详细信息",
+    "for.gui.triple": "三倍",
+    "for.gui.unknown": "未知",
+    "for.gui.warm": "温暖",
+    "for.gui.water": "水域",
+    "for.gui.worker": "工蜂",
+    "for.hints.apiaristsuit.tag": "养蜂套装！",
+    "for.hints.beebiomes.tag": "选择合适的生物群系！",
+    "for.hints.beebreeding.desc": "不同种群的雄蜂和雌蜂（公主蜂）交配可能产生新种群的后代。",
+    "for.hints.beebreeding.tag": "饲养你的蜜蜂！",
+    "for.hints.beeflowers.desc": "蜜蜂所在，鲜花飘香，有的罕见，有的寻常。",
+    "for.hints.beeflowers.tag": "蜜蜂需要鲜花",
+    "for.hints.beejubilance.tag": "取悦蜜蜂！",
+    "for.hints.cooldownloss.tag": "强制冷却能量会损耗",
+    "for.hints.crating.desc": "装箱储存可以压缩你盈余的物资的储存空间，你可以把大量的方块以以及物品以装箱后的形式成堆地储存。",
+    "for.hints.crating.tag": "装箱储存！",
+    "for.hints.energybuffer.tag": "能量缓存",
+    "for.hints.energyunit.desc": "引擎输出能量的单位为RF/t，意为“RF每tick”。每秒有20tick。",
+    "for.hints.energyunit.tag": "所谓RF(Redstone Flux，红石通量)",
+    "for.hints.enginebronzefuels.desc": "生物燃油引擎可以使用多种燃料：生物质，种子油，蜂蜜，牛奶甚至水！",
+    "for.hints.enginebronzefuels.tag": "可使用多种燃料！",
+    "for.hints.enginebronzeheatloss.tag": "用水时请小心！",
+    "for.hints.enginebronzeoutput.desc": "不同的燃料产能不同。输出范围从 10 RF/t 到 50 RF/t 不等。",
+    "for.hints.enginebronzeoutput.tag": "选择合适的燃料！",
+    "for.hints.enginebronzewaste.desc": "引擎相连并不会导致能量流失。",
+    "for.hints.enginebronzewaste.tag": "能量守恒",
+    "for.hints.enginecopperfuels.tag": "可使用多种燃料！",
+    "for.hints.enginecopperwaste.tag": "防止能量流失！",
+    "for.hints.enginetinbatteries.tag": "为电池充电。",
+    "for.hints.enginetinchoke.desc": "可以通过为电力引擎加装电子板来扼流，以减少输出(减少为10RF/t)和燃料消耗。",
+    "for.hints.enginetinchoke.tag": "调节输出",
+    "for.hints.flowerjungle.desc": "丛林花指的是蕨类或藤蔓。",
+    "for.hints.flowerjungle.tag": "什么是丛林花？",
+    "for.hints.flowernether.desc": "下界花指的是地狱疣。",
```

```suggestion
    "for.hints.flowernether.desc": "下界花指的是地狱疣。",
```
    "for.hints.flowernether.desc": "下界花指的是下界疣。",